src/app/api-server/tests/tests/test_di_graph.rs

```diff
@@ -7,29 +7,35 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use dill::*;
-
 #[test_log::test(tokio::test)]
 async fn test_di_graph_validates_local() {
     let tempdir = tempfile::tempdir().unwrap();
-    let mut catalog_builder = kamu_api_server::init_dependencies(
-        kamu_api_server::config::ApiServerConfig::default(),
-        &url::Url::from_directory_path(tempdir.path()).unwrap(),
-        false,
-        tempdir.path(),
+
+    let multi_tenant = false;
+    let config = kamu_api_server::config::ApiServerConfig::default();
+    let repo_url = url::Url::from_directory_path(tempdir.path()).unwrap();
+
+    let dependencies_graph_repository = kamu_api_server::prepare_dependencies_graph_repository(
+        kamu::domain::CurrentAccountSubject::new_test(),
+        &repo_url,
+        multi_tenant,
+        &config,
     )
     .await;
 
+    let mut catalog_builder =
+        kamu_api_server::init_dependencies(config, &repo_url, multi_tenant, tempdir.path()).await;
+
+    catalog_builder.add_value(dependencies_graph_repository);
```

Comment:
Admittedly, I couldn't think of anything better than just adding it to the check.
Since, we have the special catalog inside, this is the only way we can check for errors in it